skxray/core/image.py

```diff
@@ -97,3 +97,58 @@ def _corr_ax1(input_image):
                                          mode='full')/norm_mask) / 2
              for v in input_image]
     return np.histogram(est_by_row, bins=np.arange(0, dim + 1))
+
+def hist_make(src_data,
+              num_bins,
+              pd_function=False):
+    """
+    This function evaluates the histogram of the source data set and returns
+    bin data consisting of both bin edges and bin averages. This tool is
+    primarily geared for plotting histograms for visual analysis and
+    comparison.
+
+    Parameters
+    ----------
+    src_data : ndarray
+        Can be JxK or IxJxK
+        Specifies the source data set from which you want to evaluate the
+        histogram.
+    num_bins : int
+        Specify the number of bins to include in the histogram as an integer.
+
+    pd_function : bool, optional
+        Identify whether the histogram data should be normalized as a
+        probability density histogram or not.
+        Options:
+            True -- Histogram data is normalized to range from 0 to 1
+            False -- Histogram data reported simply as "counts" (e.g. Voxel
+                     Count)
+
+    Returns
+    -------
+    hist : array
+        1xN array containing all of the actual bin measurements (e.g. voxel
+        counts)
+    bin_avg : array
+        1xN array containing the average intensity value for each bin.
```

Comment:
Is the the average (ex, the center) of the bins or the average of the values that are put into this bin?